src/config/ConfigManager.cpp

```diff
@@ -1644,6 +1646,9 @@ void CConfigManager::loadConfigLoadVars() {
         ensureVRR();
     }
 
+    if (!isFirstLaunch)
```

Comment:
this crashes if run on 1st launch, could cause problems when run on weird states, but I'm not sure

Reply:
`&& !g_pCompositor->m_bUnsafeState`

Reply:
should I log it as a reload error?